roles/deploy_ioc/vars/adpicam.yml

```
@@ -0,0 +1,6 @@
+---
+
+deploy_ioc_use_common: true
+deploy_ioc_use_ad_common: true
+deploy_ioc_required_module: adpicam_0d86fae
+deploy_ioc_executable: ADPICamApp
```

```suggestion
deploy_ioc_executable: ADPICamApp
deploy_ioc_template_root_path:
  "{{ deploy_ioc_required_module_path }}/iocs/PICamIOC
```

You'll also need to add a list for `deploy_ioc_device_specific_env:` Copying over the components from `ioc-deploy-roles/roles/DeployIOC/vars/AdPICam.yml`, taking not of what variables can be scrapped or changed name. Copied below

```yaml
_ioc_specific_default_environment:
  ADPICAM: "/epics/modules/ADPICam"
  PORT: "PICAM1"
  QSIZE: 20
  XSIZE: 2048
  YSIZE: 2048
  NCHANS: 2048
  CBUFFS: 500
  MAX_THREADS: 8
  EPICS_DB_INCLUDE_PATH: "$(ADCORE)/db:$(ADPICAM)/db"
  EPICS_CA_MAX_ARRAY_BYTES: 5000000
```